src/main/java/io/confluent/idesidecar/websocket/messages/Audience.java

```diff
@@ -0,0 +1,33 @@
+package io.confluent.idesidecar.websocket.messages;
+
+import io.quarkus.runtime.annotations.RegisterForReflection;
+
+@RegisterForReflection
+public enum Audience {
+  /**
+   * Message intended for a single workspace, usually originating from the sidecar as a response
+   * to a message from the workspace.
```

Comment:
Before I found that the AccessTokenFilter was kicking in and discovered it would be sufficient for authorizing the connection, there was an ACCESS_REQUEST + ACCESS_RESPONSE message pair sent over the websocket connection using this request/response scheme.

It could well end up that this messaging pattern is never needed, and that all such request/response pairs should go over http/rest. Back at Noteable we had a mix, and it was hard to decide what mechanism to use sometimes. I'll remove reference for now.

Reply:
Will remove Audience.WORKSPACE at this time. Can grow it back in the future if ever needed.

Reply:
Doing away with entire header.audience field and then the Audience enum. Not needed at this time. Can always grow back in the future when needed.